Marlin/src/pins/pins.h

```diff
@@ -511,7 +511,7 @@
 #elif MB(BTT_SKR_E3_DIP)
   #include "stm32f1/pins_BTT_SKR_E3_DIP.h"      // STM32F1                                env:STM32F103RE_btt env:STM32F103RE_btt_USB env:STM32F103RC_btt env:STM32F103RC_btt_512K env:STM32F103RC_btt_USB env:STM32F103RC_btt_512K_USB
 #elif MB(BTT_SKR_CR6)
-  #include "stm32f1/pins_BTT_SKR_CR6.h"         // STM32F1                                env:STM32F103RC_btt_512K_USB
+  #include "stm32f1/pins_BTT_SKR_CR6.h"         // STM32F1                                env:STM32F103RC_btt_512K_USB env:STM32F103RE_btt_USB
```

Comment:
This boards ships with an RET6 chip (to match the original), so the PIO environments should be `STM32F103RE_btt` and `STM32F103RE_btt_USB`

Reply:
Was planning on looking into this today to determine if it was the pins.h or the test at fault. You would know, so updating pins.